packages/misc/src/lib/misc.ts

```diff
@@ -181,20 +180,14 @@ export const isNodeClientErrorV0 = (
 };
 
 export const isNodeErrorV1 = (
-  nodeError: NodeErrorV0 | NodeErrorV1
+  nodeError: NodeErrorV1
 ): nodeError is NodeErrorV1 => {
   return (
     nodeError.hasOwnProperty('errorCode') &&
     nodeError.hasOwnProperty('errorKind')
   );
 };
```

Comment:
We can remove this function too

![Screenshot from 2024-02-23 11-50-11](https://github.com/LIT-Protocol/js-sdk/assets/10857348/8c383ad8-2fc8-4e7b-a9c0-0d473a20ce01)

Reply:
good catch, removed it!